components/custom-network/theme-colors.tsx

```diff
@@ -51,16 +45,33 @@ const ThemeStringInput = ({label, text, error, onChange}) => {
           d-flex flex-row align-items-center border-radius-8  cursor-pointer
           `}
           >
-          <input
-            type="text"
-            className={`custom-text-input ${(error && "is-invalid") || ""} form-control`} 
-            name={label}
-            id={label}
-            value={text}
-            ref={themeStringInput}
-            onChange={handleChange}
-            onClick={copyContent}
-          />
+          {/* The update key is used to force a rerender on the input component when a color input changed. */}
+          <div key={updateKey}>
```

Comment:
The update key is a neat 'trick' to update the initialValue when a color Input changes.